hack/ci-test-e2e.sh

```diff
@@ -12,9 +13,17 @@ generate_junit() {
 }
 trap generate_junit EXIT
 
+if [ -z "${1:-}" ]
+then
+      echo "Running all tests"
+else
+      echo "Running tests matching ${1}"
```

Comment:
This and below also needs the variable defaulting. It might be easier to define a new `TEST_RUN` or some such var that gets the value of `${1:-}` and use that

Reply:
There is still a second reference to `$1` on line 16 :upside_down_face:

Reply:
sorry! updated.